source/includes/fact-write-blocking-enable.rst

```diff
@@ -1,7 +1,9 @@
 When write-blocking is enabled, ``mongosync`` blocks writes:
 
-- On the destination cluster during sync
-- On the source cluster while committing
+- On the destination cluster during sync.
+- On the source cluster while ``commit`` is running.
+- On the destination cluster while ``commit`` is running, until
```

Comment:
To be clear what I meant was move the entire commit section below the write block section. This way the concept of write blocking is introduced first and then the commit content is introduced. 

I think combining them is slightly confusing, as these bullets are contradicting:

- On the destination cluster during sync.
- On the destination cluster while ``commit`` is running, until ``mongosync`` begins index validation.

Reply:
ah my bad -  I moved the commit section below write-blocking and uncombined the two!